src/components/AnnotationTab/AnnotationTab.js

```diff
@@ -1,65 +0,0 @@
-import { observer } from 'mobx-react';
-import { Block, Elem } from '../../utils/bem';
-import { CurrentEntity } from '../CurrentEntity/CurrentEntity';
-import Entities from '../Entities/Entities';
-import Entity from '../Entity/Entity';
-import Relations from '../Relations/Relations';
-import { Comments } from '../Comments/Comments';
```

Comment:
All this stuff has also to be removed if not in use. By shallow check that's `CurrentEntity` and `Entity` at least.